native/thrift_struct_read_module.h

```diff
@@ -0,0 +1,15 @@
+#ifndef THRIFT_STRUCT_READ_MODULE_H
```

Comment:
the second line should be `#define THRIFT_STRUCT_READ_MODULE_H`

see: https://en.wikipedia.org/wiki/Include_guard